src/less/button/button.less

```diff
@@ -273,13 +273,7 @@ a.fake-btn--secondary[aria-disabled="true"] {
 
 button.btn--tertiary,
 a.fake-btn--tertiary {
-    .background-color-token(btn-tertiary-background-color, color-background-secondary);
-    .border-color-token(btn-tertiary-border-color, color-background-secondary);
-    .color-token(btn-tertiary-foreground-color, color-foreground-accent);
-}
-
-a.fake-btn--tertiary:visited {
-    .color-token(btn-tertiary-foreground-color, color-foreground-accent);
+    .border-color-token(btn-tertiary-border-color, color-stroke-default);
```

Comment:
Hmm...we're getting rid of background and color for both standard and dark mode? 

1. Are these buttons always going to inherit color? 
2. Will they have a transparent background by default? 

Let's verify these with the design team.

Reply:
So as per figma, yes that's what is intended. This looks correct. This is the same as we have currently with secondary button.